lua/astronvim/utils/status/provider.lua

```diff
@@ -288,13 +288,14 @@ end
 -- @see astronvim.utils.status.utils.stylize
 function M.filename(opts)
   opts = extend_tbl({
-    fallback = "Empty",
+    fallback = "[No Name]",
```

Comment:
```suggestion
    fallback = "Empty",
```

Please revert this part as I would like to leave this as "Empty". It's not saying the buffer is empty, but the name is empty. Also it saves room. You can override this in your user configuration if you prefer "[No Name]"